squad-server/plugins/basic_stats.js

```diff
@@ -0,0 +1,738 @@
+/* eslint-disable no-duplicate-case */
+/* eslint-disable no-dupe-class-members */
```

Comment:
As with the above, I think this can be removed without issue. This flag prevents multiple methods of the same name which has not been done.